React/game/src/app/screens/Login/components/Login/layout.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Field, reduxForm } from 'redux-form';
+
+import styles from './styles.scss';
+
+function LoginLayout(props) {
+  const { handleSubmit } = props;
+  return (
+    <form onSubmit={handleSubmit}>
+      <div className={styles.login}>
+        <div>
```

Comment:
are we using this div for anything?

Reply:
Also, remember to use PropTypes in every component

Reply:
👍